UACloudLibraryServer/Controllers/InfoModelController.cs

```diff
@@ -187,9 +187,14 @@ public IActionResult GetAllNamesandAddressSpacesAsync()
                 return new ObjectResult("Contributor name of existing nodeset is different to the one provided.") { StatusCode = (int)HttpStatusCode.Conflict };
             }
 
+            uint nodeSetHashCodeToStore = nodesetHashCode;
```

Comment:
This is dangerous as you're giving overwrite control to the client. We explicitly want to NOT read the identifier from the client, even if the client specifies one. Also, we want to migrate away from the legacy hashing function as the new one is much better. Please revert this change.

Reply:
Looking into this more, there are better ways to optimize the sync. I'll revert this change.